app/models/AppealData.scala

```diff
@@ -16,20 +16,19 @@
 
 package models
 
+import java.time.LocalDate
 
-import models.v2.AppealInformation
 import play.api.libs.json.{Format, Json}
 
-import java.time.LocalDateTime
-
 case class AppealData[A](
                        `type`: PenaltyTypeEnum.Value,
-                        startDate: A,
-                        endDate: A,
-                        dueDate: A,
-                        dateCommunicationSent: A
-                     ) extends AppealInformation[A]
+                       startDate: A,
+                       endDate: A,
+                       dueDate: A,
+                       dateCommunicationSent: A
+                     )
 
 object AppealData {
-  implicit val format: Format[AppealData[LocalDateTime]] = Json.format[AppealData[LocalDateTime]]
+  implicit val format: Format[AppealData[LocalDate]] = Json.format[AppealData[LocalDate]]
```

Comment:
We can revert this back to how it was before, by removing the type parameter ([A]) from the case class and changing startDate: A to startDate: LocalDate etc.